e2e_tests/cloud.py

```diff
@@ -5,5 +5,5 @@ def get_cloud() -> cloud.Cloud:
     return cloud.get_active_cloud()
 
 
-def get_aad_authority_url() -> str:
-    return get_cloud().endpoints.active_directory
+def get_authority_domain() -> str:
```

Comment:
Can you verify this is the same method name we used in the other apps?

Reply:
Yes, the exact same one as in the api, I just added the 'str' for the return value to remain consistent

Reply:
FIixed